documentation/docs/examples/cad-colors.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: cad-colors
+title: Coloring CAD models
+---
+
+import { LiveCodeSnippet }from '../../src/components/LiveCodeSnippet';
+import { DemoWrapper } from '../../src/components/DemoWrapper';
+import { Cognite3DModel, Cognite3DViewer } from '@cognite/reveal';
+
+<DemoWrapper modelId={4715379429968321} revisionId={5688854005909501} name="Cognite3DViewerDemo" />
+
+Reveal supports updating the color of individual objects by providing [tree indices to identify 3D nodes](../concepts.md).
+
+### Update all node colors
+To update the colors of all nodes, loop over all tree indices in the model and update the colors
+accordingly:
+
+<LiveCodeSnippet>
+{`
+model.iterateNodesByTreeIndex(treeIndex => {
+  model.setNodeColorByTreeIndex(treeIndex, 127, 127, 127);
+});
+`}
+</LiveCodeSnippet>
+
+### Reset node colors
+
+Color overrides can be overridden using `Cognite3DModel.resetNodeColorByTreeIndex`, e.g.
+<LiveCodeSnippet>
+{`
+model.iterateNodesByTreeIndex(treeIndex => {
+  model.resetNodeColorByTreeIndex(treeIndex);
+});
+`}
+</LiveCodeSnippet>
+
+It's also possible (and more performant) to use `Cognite3DModel.resetAllNodeColors()` when
+you want to reset the colors of all nodes:
+<LiveCodeSnippet>
+{`
+model.resetAllNodeColors();
+`}
+</LiveCodeSnippet>
+
+### Update the colors of a subtree
+The 3D nodes are organized in a tree structure, where e.g. equipment will have many sub-components. In 
+order to update the colors of equipment it's typically necessary to apply the same color to the entire
+subtree. Reveal supports this through the use of `Cognite3DModel.setNodesColorsByTreeIndex` with `applyToChildren`
```

Comment:
TBH I can't notice how it works. The model itself has some colors already, and when I click run, I have no idea where my subtree is and what changed. Perhaps we should just add here this line: 
```js
model.iterateNodesByTreeIndex(treeIndex => {
  model.setNodeColorByTreeIndex(treeIndex, 127, 127, 127);
});
```
 to make setNodeColorByTreeIndex visible

I tried to do that but still can't see the subtree that changed 🤷‍♂️  Maybe we can to point camera on that subtree somehow?